source/reference.txt

```diff
@@ -1,9 +1,53 @@
-.. place holder for the reference section TOC
-
 =========
 Reference
 =========
 
+:ref:`c2c-mongosync`
+   The primary process used in {+c2c-product-name+}.
+
+:ref:`c2c-config`
+   Configure :program:`mongosync` instances at startup using a
+   configuration file.
+
+:ref:`c2c-api`
+   The API endpoints to interact with ``mongosync``. 
+
+:ref:`c2c-states`
+   States that ``mongosync`` enters depending on requests received. 
+
+:ref:`c2c-filtered-sync`
+   Inclusion filters to specify which databases and collections to
+   include in sync. 
+
+:ref:`c2c-oplog-sizing`
+    Use :ref:`change streams <changeStreams>` to synchronize data
+    between source and destination clusters.
+
+:ref:`c2c-failure-recovery`
+   Prepare for disaster recovery using :ref:`connections
+   <c2c-connecting>` between clusters.
+   
+:ref:`c2c-limitations`
+   Limitations of ``mongosync``. 
+
+:ref:`c2c-logging`
+   A running log of events, including entries such as the copying
+   collections and indexes, change stream events, and API  calls. 
```

Comment:
[nit]

Consider removing the additional space between "API" and "calls" on line 35.

Reply:
Done!